controllers/territory_controller/territory_controller.py

```diff
@@ -57,10 +66,12 @@ def get_claimant(
         return self._station_statuses[station_code]
 
     def setup(self) -> None:
-        self._emitters = {station_code: self._robot.getEmitter(station_code + "Emitter")
+        self._emitters = {station_code:
+                          get_robot_emitter(self._robot, station_code + "Emitter")
                           for station_code in StationCode}
```

Comment:
nit: I think I'd expect this to be wrapped like:
```suggestion
        self._emitters = {
            station_code: get_robot_emitter(self._robot, station_code + "Emitter")
            for station_code in StationCode
        }
```
I realise that's not quite like the other comprehensions in this file, though that's a result of this file being out of step with the rest of this repo. (Unfortunately we don't have a linter for this yet)

(also `_receivers` below)